tfx/proto/trainer.proto

```diff
@@ -17,16 +17,22 @@ package tfx.components.trainer;
 
 // Args specific to training.
 message TrainArgs {
+  // The example split names to train on.
+  string splits = 1;
```

Comment:
it's repeated string (list of splits), same for eval

Reply:
got it, i changed it, thanks for the catch